warden/x/act/keeper/abci.go

```diff
@@ -3,24 +3,44 @@ package keeper
 import (
 	"context"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"time"
 )
 
 func (k Keeper) EndBlocker(ctx context.Context) error {
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 	blockHeight := sdkCtx.BlockHeight()
 	params := k.GetParams(ctx)
 
+	if params.MaxPendingTime > 0 && params.MaxCompletedTime > 0 {
```

Comment:
@Pitasi, tbh, I’m not sure what the right way is to add parameters to a module that didn’t have them before. The issue is that the generated snapshots don’t store the parameters, so they’re being initialized with default (0) values. 
I’d appreciate it if you could give me some direction on how to set default values during initialization from the genesis.

Reply:
I see that `genesis.json` contains the default params correctly:

```json
    "act": {
      "params": {
        "max_pending_time": "86400s",
        "max_completed_time": "604800s",
        "prune_check_block_frequency": "10000"
      },
      "actions": [],
      "rules": []
    },
```

In `x/act/module/genesis.go` there is the `InitGenesis` function the is invoked the first time the chain starts, reading the genesis file. Inside, it calls `k.SetParams()` using the params contained in the genesis file.

Everything seems correct, in fact you can launch localnet and query the params:
```
$ wardend q act params
params:
  max_completed_time: 168h0m0s
  max_pending_time: 24h0m0s
  prune_check_block_frequency: "10000"
```

I'll continue investigating why it wasn't working before.

Reply:
https://github.com/warden-protocol/wardenprotocol/blob/main/tests/testdata/snapshot-base/config/genesis.json

Reply:
i'm gonna merge this, then in #800 i regenerated the snapshots correctly